src/com/zimbra/webClient/servlet/ServiceServlet.java

```diff
@@ -118,6 +124,15 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
                 } else if ("/publiclogin".equals(path)) {
                     //this operation loads a JSP with login form for public login.
                     doPublicLoginProv(req, resp);
+                } else if ("/flushall".equals(path)) {
```

Comment:
Is **flushall** a superset of **flushuistrings** and **flushzimlets**?  If not then I thing something like **flushpermcache** would make it clearer what the API does.